vanilla-tests/src/main/java/fixtures/streamstylexmlserialization/implementation/models/BananaWrapper.java

```diff
@@ -65,9 +66,9 @@ public static BananaWrapper fromXml(XmlReader xmlReader, String rootElementName)
                     List<Banana> items = null;
 
                     while (reader.nextElement() != XmlToken.END_ELEMENT) {
-                        String elementName = reader.getElementName().getLocalPart();
+                        QName elementName = reader.getElementName();
 
-                        if ("banana".equals(elementName)) {
+                        if ("banana".equals(elementName.getLocalPart())) {
```

Comment:
nit, seems equivalent before/after change?

Reply:
Yeah, this is equivalent before and after but using `QName` allows for the namespace to be checked as well.